package.json

```diff
@@ -68,6 +68,7 @@
     "@tippyjs/react": "^4.2.6",
     "clsx": "^1.1.1",
     "downshift": "^6.1.7",
+    "pascal-case": "^3.1.2",
```

Comment:
I tried to add this in dev dependencies but it complained 😕

Reply:
oo was it a lint error? within `.eslintrc`, in the `"import/no-extraneous-dependencies"` rule specifically, you may need to add `plopfile.js` to the `"devDependencies" array.

that error is making sure we don't accidentally use a dev dependency in something that will be published, so just need to make sure we've told lint all the files that won't ever be published

Reply:
Awesome, thanks!